### PR TITLE
Fixed bug where property value was showing up as '$undefined'

### LINKED ISSUE #2: Assessed value is showing as undefined
Assessed value is showing `undefined`, the network request to the API is showing the correct value (Dev tools).

Looks like the data set changed the API_Field Name from `total_asmt` to `assessed_value`.

https://data.edmonton.ca/City-Administration/Property-Assessment-Data-Current-Calendar-Year-/q7d6-ambg

### Fix
Update the property field name to the new value in `property.model.ts`

### PR DESCRIPTION
**Pull request by WonHan7**

### CODE REVIEW

File: src/app/core/assessment.model.ts [hunk 1]
```diff
@@ -11,4 +11,5 @@ export class Assessment {
   public garage: string;
   public latitude: number;
   public longitude: number;
+  public assessed_value: number;
```
Comment (linhub15) [inline src/app/core/assessment.model.ts:13]: If there are no references to it, can we remove the `total_asmt` on line 6?

File: src/app/core/property.model.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export class Property {
   }
 
   get assessedValue(): string {
-    return `$${this._assessment.total_asmt}`;
+    return `${this._assessment.assessed_value}`;
```
Comment (linhub15) [inline src/app/core/property.model.ts:25]: Does removing one of the dollar symbols `$` still display the dollar symbol on the search component?

**Review by linhub15 — CHANGES_REQUESTED** [review]

File: src/app/core/assessment.model.ts (hunk 1 above)
Comment (WonHan7) [reply src/app/core/assessment.model.ts:13]: Yes it should be safe to remove total_asmt from assessment.model.ts.

File: src/app/core/property.model.ts (hunk 2 above)
Comment (WonHan7) [reply src/app/core/property.model.ts:25]: The first dollar symbol was being read as a string dollar symbol which is where the $ in $undefined was coming from.

**Comment by linhub15** [comment]
Looks good to me!

**Review by linhub15 — APPROVED** [review]

**Pull request merged**